src/components/breadcrumb/_breadcrumb.scss

```diff
@@ -29,6 +29,10 @@
     color: $text-03;
   }
 
+  .bx--breadcrumb__last__item::after{
```

Comment:
Should I update with:
`.bx--breadcrumb-item__without-trailing-slash .bx--breadcrumb-item:last-child::after {
  content: '';
}`
Or keep it as 
` .bx--breadcrumb-item--last::after{
    content: '';
  }`
@asudoh @hellobrian